README.md

```diff
@@ -72,3 +72,30 @@ For full details on how to use Streamlink visit our documentation at [streamlink
 Feel free to open a bug or contribute to code!
 
 No need to go through a large CONTRIBUTING.md doc. The only requirement being that we get at least one ACK from a maintainer. Fork / clone us and open a PR!
+
+<a href="graphs/contributors"><img src="https://opencollective.com/streamlink/contributors.svg?width=890" /></a>
```

Comment:
I'd also like to keep this in the Contributing section (as I see financial contributions, answering questions, troubleshooting, and so forth all of great value and we should probably update the README to reflect that). I also agree on the usage of HTML tags here and below in the `Backers` section. As it's just an image we can easily include it via markdown and it keeps the document from getting confusing.